traefik/values.yaml

```diff
@@ -935,6 +935,8 @@ hub:
   # -- Name of `Secret` with key 'token' set to a valid license token.
   # It enables API Gateway.
   token: ""
+  # -- Array of namespaces to watch. If left empty, Traefik watches all namespaces.
+  namespaces: []
```

Comment:
There are already [namespaces in kubernetes CRD](https://doc.traefik.io/traefik/providers/kubernetes-crd/#namespaces), [namespaces in kubernetes Ingress](https://doc.traefik.io/traefik/providers/kubernetes-ingress/#namespaces) and [namespaces in kubernetes Gateway](https://doc.traefik.io/traefik/providers/kubernetes-gateway/#namespaces)

Since hub provider works with all k8s providers, wdyt of adding a warning in NOTES.txt when namespaces does not match ?

Reply:
Yes that's a good idea!